serverless/database-backup/src/backup.ts

```diff
@@ -0,0 +1,102 @@
+import path from 'path'
+import AWS from 'aws-sdk'
+
+import config from './config'
+import Encryptor from './encryptor'
+import { PgDump, SecretsManagerDump } from './dumps'
+import { configureEndpoint } from './utils/aws-endpoint'
+
+const S3 = new AWS.S3({
+  ...configureEndpoint(config),
+  computeChecksums: true,
+})
+
+class Backup {
+  pgDump: PgDump
+  secretsDump: SecretsManagerDump
+  encryptor: Encryptor
+
+  constructor(options: {
+    pgDump: PgDump
+    secretsDump: SecretsManagerDump
+    encryptor: Encryptor
+  }) {
+    const { pgDump, secretsDump, encryptor } = options
+    this.pgDump = pgDump
+    this.encryptor = encryptor
+    this.secretsDump = secretsDump
+  }
+
+  private getBackupFolderName(): string {
+    const today = new Date()
+    const yyyy = today.getFullYear()
+    const dd = today.getDate().toString().padStart(2, '0')
+    const mm = (today.getMonth() + 1).toString().padStart(2, '0')
+
+    return `${yyyy}-${mm}-${dd}`
```

Comment:
does the timezone matter...? will this always be in utc / match with the date of the rds snapshot? what's wrong with `(new Date()).toISOString().substring(0,10)`

Reply:
Time in lambda runtime will be UTC. The `TZ` env var in the runtime is [fixed to UTC](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html#configuration-envvars-runtime). Backup window for RDS is also defined in UTC.

Will update the generation of the folder name to the one-liner! Thanks! 👍